.github/workflows/gradle.yml

```diff
@@ -247,18 +247,8 @@ jobs:
       #      - name: Get Docker Space
       #        run: docker run --rm busybox df -h
 
-      - name: Image Cleanup
-        run: ./tools/bin/clean_images.sh
-
-      - name: Build Platform Docker Images
-        if: success() && github.ref == 'refs/heads/master'
-        run: GOOGLE_APPLICATION_CREDENTIALS="/tmp/gcs.json" ./gradlew composeBuild --scan
-        env:
-          GIT_REVISION: ${{ github.sha }}
-
       # make sure these always run before pushing platform docker images
       - name: Run End-to-End Acceptance Tests
-        if: success() && github.ref == 'refs/heads/master'
```

Comment:
we run k8s acceptance tests as part of PR. no reason we shouldn't do it for docker. also doing so would have cost the dependencies i missed in my PR.

Reply:
definitely. surprised this aren't running always.